renderer/reducers/backup.js

```diff
@@ -224,6 +274,54 @@ export const setBackupPathLocal = localPath => {
     localPath,
   }
 }
+/**
+ * setRestoreMode - sets if we are in the restore mode currently.
+ *
+ * @param {boolean} value local filesystem directory URI
```

Comment:
`local filesystem directory URI` seems like the wrong description for this

Reply:
done